internal/course/delivery/http/course_handler.go

```diff
@@ -64,26 +63,26 @@ func (c *CourseHandler) GetAll(echoContext echo.Context) error {
 		case "start":
 			val := strings.TrimSpace(v[0])
 			if start, err = strconv.Atoi(val); err != nil {
-				return echoContext.JSON(getStatusCode(err), ResponseError{Message: err.Error()})
+				return echoContext.JSON(util.GetStatusCode(err), ResponseError{Message: err.Error()})
 			}
 		case "limit":
 			val := strings.TrimSpace(v[0])
 			if limit, err = strconv.Atoi(val); err != nil {
-				return echoContext.JSON(getStatusCode(err), ResponseError{Message: err.Error()})
+				return echoContext.JSON(util.GetStatusCode(err), ResponseError{Message: err.Error()})
 			}
 		}
 	}
 
 	listCourse, err := c.CourseUseCase.GetAll(ctx, start, limit)
 	if err != nil {
-		return echoContext.JSON(getStatusCode(err), ResponseError{Message: err.Error()})
+		return echoContext.JSON(util.GetStatusCode(err), ResponseError{Message: err.Error()})
 	}
 	return echoContext.JSON(http.StatusOK, listCourse)
 }
 
 // GetByID ...
 func (c *CourseHandler) GetByID(echoContext echo.Context) error {
-	fmt.Println("Calling GetByID Courses")
+	log.Info("Calling GetByID Courses")
```

Comment:
`log.Info` is expensive to me since we do not log with any other information but notice that we are calling a method.